config/routes.rb

```diff
@@ -4,6 +4,6 @@
   get 'about'       => 'pages#about'
   get 'contact'     => 'pages#contact'
   get 'volunteer'   => 'pages#volunteer'
-  get 'adopt'       => 'pages#adopt'
+  get 'adopt'       => 'rat_adoptions#index'
```

Comment:
Now that your using a restful route, how would you feel about using the rails helper for this, i.e.

```rb
resources :rat_adoptions, only: [:index]
```